service/storage_proxy.cc

```diff
@@ -990,6 +996,12 @@ class storage_proxy::remote {
 
         auto schema = co_await get_schema_for_read(cmd.schema_version, src_addr, src_shard, *timeout);
         dht::token token = dht::get_token(*schema, key);
+
+        // This guard ensures that tablet migrations wait for replica requests,
+        // even if the LWT coordinator has been fenced out.
```

Comment:
Why do you call it "fenced out"? Wouldn't it be better to say "even if the LWT query already completed on the coordinator"?